sdks/python/container/run_generate_requirements.sh

```diff
@@ -104,4 +104,13 @@ EOT
 # Remove pkg_resources to guard against
 # https://stackoverflow.com/questions/39577984/what-is-pkg-resources-0-0-0-in-output-of-pip-freeze-command
 pip freeze | grep -v pkg_resources >> "$REQUIREMENTS_FILE"
+
+if grep -q "tensorflow==" "$REQUIREMENTS_FILE"; then
```

Comment:
i recommend to move changes to this file into a separate PR.

Reply:
Done. https://github.com/apache/beam/pull/27789